nilearn/decoding/space_net.py

```diff
@@ -630,6 +630,12 @@ class BaseSpaceNet(CacheMixin, LinearRegression):
     debias : :obj:`bool`, optional (default False)
         If set, then the estimated weights maps will be debiased.
 
+    positive : bool, default=False
+        When set to ``True``, forces the coefficients to be positive.
+        This option is only supported for dense arrays.
+
+        .. versionadded:: 0.11.2dev
```

Comment:
```suggestion
        .. versionadded:: 0.11.2
```
`dev` shouldn't be there?